docs/content/tutorials/kic/ingress-walkthrough.md

```diff
@@ -115,14 +115,14 @@ kind: Ingress
 metadata:
   name: bookinfo-ingress
 spec:
-  # ingressClassName: nginx # use only with k8s version >= 1.18.0
+  ingressClassName: nginx # use only with k8s version >= 1.18.0
```

Comment:
There are also a couple of files in our `examples/` directory that have the same issues that you corrected here. Would you mind updating those as well?

Reply:
Hmm, yeah let me check those out...